internal/servicedeployer/custom_agent.go

```diff
@@ -94,7 +94,7 @@ func (d *CustomAgentDeployer) SetUp(ctx context.Context, svcInfo ServiceInfo) (D
 		fmt.Sprintf("%s=%s", fleetPolicyEnv, d.policyName),
 	)
 
-	configDir, err := d.installDockerfile()
+	configDir, err := d.installDockerfile(deployerFolderName(svcInfo))
```

Comment:
I wonder if we could use go-resource to manage these files.

Reply:
Sure, updated `custom_agent` to create the required files using go-resource.

Terraform (servicedeployer) was already using `go-resource` to create the required configuration files or scripts.